app/form_builders/adp_form_builder.rb

```diff
@@ -45,7 +45,7 @@ def full_anchor_name_for(_object, anchor_name)
 
   def make_object_name
     klass_name = object.class.to_s
-    klass_name = 'Claim' if klass_name =~ /^Claim::/
+    klass_name = 'Claim' if klass_name.match?(/^Claim::/)
```

Comment:
🤔 Was this a cop suggestion? - i ask because match? is clearer but =~ is more performant from my understanding. 
...
Just noticed the question mark  - match? - looks like introduced in 2.4 regexp and assume it is performant 👍